www/src/data/sidebars/doc-links.yaml

```diff
@@ -189,17 +185,23 @@
           link: /docs/creating-a-sitemap/
         - title: Linking between pages*
           link: /docs/linking-between-pages/
-        - title: Search Engine Optimization (SEO)
-          link: /docs/seo/
-    - title: Adding third-party services
-      link: /docs/adding-third-party-services/
-      items:
-        - title: Adding search
-          link: /docs/adding-search/
-        - title: Adding analytics*
-          link: /docs/adding-analytics/
-        - title: Adding forms*
-          link: /docs/adding-forms/
+        - title: Authentication*
+          link: /docs/authentication/
+        - title: E-commerce*
+          link: /docs/e-commerce/
+        - title: Interactive pages*
+          link: /docs/interactive-pages/
+        - title: Routing
+          link: /docs/routing/
+          items:
+          - title: @reach/router and Gatsby
```

Comment:
Heads up, the linter complained about this. Think you just need to wrap it in double quotes!

```
✖ bad indentation of a sequence entry at line 197, column 20:
              - title: @reach/router and Gatsby
```

Reply:
I've jumped in & added quotes to fix this.